examples/ui.content/src/content/jcr_root/content/core-components-examples/library/.content.xml

```diff
@@ -8,6 +8,7 @@
         jcr:description="AEM Core Components - Component Library"
         jcr:primaryType="cq:PageContent"
         jcr:title="Component Library"
+        sling:configRef="/conf/core-components-examples/content"
```

Comment:
Should be at the same location as the context aware config of the data layer.

Reply:
Addressed in 93a47ba